src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -2185,25 +2186,26 @@ def installDatabase(self, dbName):
       if self.exitOnError:
         DIRAC.exit(-1)
       return result
+    gLogger.debug("CREATE DATABASES : OK")
 
     perms = "SELECT,INSERT,LOCK TABLES,UPDATE,DELETE,CREATE,DROP,ALTER,REFERENCES," \
             "CREATE VIEW,SHOW VIEW,INDEX,TRIGGER,ALTER ROUTINE,CREATE ROUTINE"
-    for cmd in ["GRANT %s ON `%s`.* TO '%s'@'localhost'" % (perms, dbName, self.mysqlUser),
-                "GRANT %s ON `%s`.* TO '%s'@'%s'" % (perms, dbName, self.mysqlUser, self.mysqlHost),
-                "GRANT %s ON `%s`.* TO '%s'@'%%'" % (perms, dbName, self.mysqlUser)]:
-      result = self.execMySQL(cmd)
-      if not result['OK']:
-        error = "Error executing '%s'" % cmd
-        gLogger.error(error, result['Message'])
-        if self.exitOnError:
-          DIRAC.exit(-1)
-        return S_ERROR(error)
+    cmd = "GRANT %s ON `%s`.* TO '%s'@'%%'" % (perms, dbName, self.mysqlUser)
```

Comment:
Ouh the nice SQL injection :-)

Reply:
Oh, I didn't add it now :)

Anyway, I can "fix" it.